contrib/oc-get-customization.sh

```diff
@@ -0,0 +1,44 @@
+# shellcheck shell=bash
+#
+# This file is an example of how you might set up your environment to run the
+# console against an OpenShift cluster during development. To use it for
+# running bridge, do
+#
+# . contrib/oc-environment.sh
+# . contrib/oc-get-customization.sh
+# ./bin/bridge
+#
+# You'll need a working oc, and you'll need yq installed and in your path for
+# this script to work correctly. See:
+# - https://github.com/openshift/oc
+# - https://github.com/mikefarah/yq#install
+#
+# This will use the first secret it finds in the default namespace. All secrets
+# must be valid or removed from the namespace.
+#
+# The environment variables beginning with "BRIDGE_" act just like bridge
+# command line arguments - in fact. to get more information about any of them,
+# you can run ./bin/bridge --help
+
+LOCAL_FILE="cluster-consoleconfig-customization.yaml"
+
+if ! which yq > /dev/null; then
+    echo "yq cli not found, please checkout https://github.com/mikefarah/yq#install"
+    exit 1
+fi
+
+consoleConfig=$(
+    oc get -n openshift-console configmaps console-config -o 'template={{index .data "console-config.yaml"}}'
```

Comment:
I'd think we'd want to read the console config custom resource directly instead of using the config map. If you read the custom resource and you  it as JSON, you can use `jq` instead of `yq` which is more likely to be installed and already a prereq.

Reply:
Yeah thats also cooler because you will get the data right after changing the console config and don't need to wait for a new deployment. Expected you want use the ConfigMap, will change this.

Reply:
@spadgett I fetch customization now via `oc get consoles.operator.openshift.io cluster -o yaml` 

I stay with yq for now, I didn't find a clean way to convert the extracted data with jq to yaml. And we need yaml because the bridge reads this ConsoleConfig as yaml. Wdyt?